frontend/index.html

```diff
@@ -1,33 +1,47 @@
 <!DOCTYPE html>
 <html lang="tr">
+  <head>
+    <meta http-equiv="Content-Security-Policy" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Rythm Technologies" />
+    <title>%VITE_APP_TITLE%</title>
```

Comment:
that will be functional or ?

Reply:
Yes, it will be on other pages

Reply:
i got it! gj i saw your [dcce709](https://github.com/RythmTechnologies/congenial-dollop/pull/53/commits/dcce7091b330c520a3e1a62074adfc3f47238086) last commit!